tool/random_tbl_ops_2.py

```diff
@@ -280,6 +408,8 @@ def make_parser() -> ArgumentParser:
     parser.add_argument(
         '-r', '--read-only-workers', type=int, default=0, help='Number of read-only workers (default: 0)'
     )
+    parser.add_argument('--exclude', nargs='+', type=str, help='List of operations to exclude')
```

Comment:
Maybe also have an --ops parameter, so I can select what I want to run? Would also be nice to specify relative frequencies here.